libbeat/api/make_listener_windows.go

```diff
@@ -0,0 +1,54 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+//+build windows
+
+package api
+
+import (
+	"fmt"
+	"net"
+
+	"github.com/pkg/errors"
+
+	"github.com/elastic/beats/libbeat/api/npipe"
+)
+
+func makeListener(host string, port int) (net.Listener, error) {
+	if npipe.IsNPipe(host) {
+		pipe := npipe.TransformFromString(host)
+		sd, err := npipe.DefaultSD()
+		if err != nil {
+			return nil, errors.Wrap(err, "cannot generate security descriptor for the named pipe")
+		}
+		return npipe.NewListener(pipe, sd)
+	}
+
+	network, path, err := parse(host, port)
+	if err != nil {
+		return nil, err
+	}
+
+	if network == "unix" {
+		return nil, fmt.Errorf(
+			"cannot use %s as the host, unix sockets are only supported on Linux and macOS platforms",
```

Comment:
From build/usage of APIs it looks like unix sockets will be supported by other platforma as well.
Let's change the error message to:

"unix sockets are not supported on Windows, use npipe instead".